lib/scss-preprocessor/broccoli-import-theme-components.js

```diff
@@ -40,4 +49,23 @@ Import.prototype.addImports = function(content) {
   return content + importsString;
 };
 
+Import.prototype.addThemeComponentClasses = function(content) {
+  var inputPath = this.inputPaths[0];
+  var importFiles = walkSync(inputPath, { globs: this.include });
```

Comment:
I'd like to be able to grab all an array of all of the parent theme (ui-base-theme)'s components and merge it with the array of components defined by the sub theme.